indico/modules/events/abstracts/clone.py

```diff
@@ -25,8 +25,16 @@ class AbstractSettingsCloner(EventCloner):
     def is_visible(self):
         return self.old_event.type_ == EventType.conference
 
-    def has_conflicts(self, target_event):
-        return bool(target_event.abstract_review_questions) or bool(target_event.abstract_email_templates)
+    def get_conflicts(self, target_event):
+        conflicts = []
+
+        if bool(target_event.abstract_review_questions):
```

Comment:
You do not need the `bool()` here; the old code only used it to avoid returning an actual list instead of a bool. But that's not needed when just checking for truthiness.

Reply:
Right, I didn't realize why the `bool`s were there..